src/public/interfaces.ts

```diff
@@ -397,10 +397,38 @@ export interface OpenConversationRequest {
   /**
   * A function that is called once the conversation Id has been created
   */
-  onStartConversation?: (subEntityId: string, conversationId: string) => void;
+  onStartConversation?: (conversationSubEntityCallback: ConversationSubEntityCallback) => void;
 
   /**
   * A function that is called if the pane is closed
   */
-  onCloseConversation?: (subEntityId: string, conversationId?: string) => void;
+  onCloseConversation?: (conversationSubEntityCallback: ConversationSubEntityCallback) => void;
+}
+
+/**
+ * @private
+ * Hide from docs.
+ * ------
+*/
+export interface ConversationSubEntityCallback {
```

Comment:
A callback is a function that gets invoked when an async operation completes. This is not a callback. Consider naming it something like `OpenConversationResponse` or something along those lines.

Reply:
may be ConversationInfo or ConversationResponse ? something generic that we can use in onStartConversation and onCloseConversation

Reply:
Changed it to ConversationSubEntityResponse

Reply:
But why "ConversationSubEntity"? That name isn't mentioned anywhere in the SDK library? It's only an internal name we use for this feature.

Reply:
Yeah you are right. When I was making this change, I felt that there should be a way to distinguish this from regular conversation in a tab. The tab conversation that we had before this feature.

But since I already had the previous object  named as ConversationRequest, it makes sense to just change it to ConversationResponse.